src/lazy/expanded/compiler.rs

```diff
@@ -843,7 +843,7 @@ impl TemplateCompiler {
 
     /// Adds a `lazy_sexp` that has been determined to represent a macro invocation to the
     /// TemplateBody.
-    fn compile_macro<'top, D: Decoder>(
+    fn compile_macro_invocation<'top, D: Decoder>(
```

Comment:
🪧 I renamed this method for clarity. The entire module is dedicated to compiling macros. This particular method is compiling a TDL macro invocation within a template.